appliances/Community/Lithops/appliance.sh

```diff
@@ -0,0 +1,255 @@
+
+    # packages
+    install_deps ${DEP_PKGS} ${DEP_PIP}
+
+    # docker
+    install_docker
+
+    # Lithops
+    install_lithops
+
+    # create Lithops config file in /etc/lithops
+    create_lithops_config
+
+    # service metadata
+    create_one_service_metadata
+
+    # cleanup
+    postinstall_cleanup
+
+    msg info "INSTALLATION FINISHED"
+
+    return 0
+}
+
+service_configure()
+{
+    # update Lithops config file if non-default options are set
+    update_lithops_config
+
+    local_ca_folder="/usr/local/share/ca-certificates/minio"
+    if [[ ! -z "${ONE_APP_MINIO_ENDPOINT_CERT}" ]] && [[ ! -f "${local_ca_folder}/ca.crt" ]]; then
+        msg info "Adding trust CA for MinIO endpoint"
+
+        if [[ ! -d "${local_ca_folder}" ]]; then
+            msg info "Create folder ${local_ca_folder}"
+            mkdir "${local_ca_folder}"
+        fi
+
+        msg info "Create CA file and update certificates"
+        echo ${ONE_APP_MINIO_ENDPOINT_CERT} | base64 --decode >> ${local_ca_folder}/ca.crt
+        update-ca-certificates
+    fi
+
+    return 0
+}
+
+service_bootstrap()
+{
+    update_lithops_config
+    return 0
+}
+
+###############################################################################
+###############################################################################
+###############################################################################
+
+#
+# functions
+#
+
+install_deps()
+{
+    msg info "Run apt-get update"
+    apt-get update
+
+    msg info "Install required packages for Lithops"
+    if ! apt-get install -y "${1}" ; then
+        msg error "Package(s) installation failed: ${1}"
+        exit 1
+    fi
+
+    msg info "Install pip dependencies"
+    if ! pip install "${2}" ; then
+        msg error "Python pip dependencies installation failed"
+        exit 1
+    fi
+}
+
+install_docker()
+{
+    msg info "Add Docker official GPG key"
+    install -m 0755 -d /etc/apt/keyrings
+
+    curl -fsSL https://download.docker.com/linux/ubuntu/gpg -o /etc/apt/keyrings/docker.asc
+
+    chmod a+r /etc/apt/keyrings/docker.asc
+
+    msg info "Add Docker repository to apt sources"
+    echo "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.asc] https://download.docker.com/linux/ubuntu \
+        $(. /etc/os-release && echo "$VERSION_CODENAME") stable" | \
+        sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
+    apt update
+
+    msg info "Install Docker Engine"
+    if ! apt-get install -y docker-ce=$DOCKER_VERSION docker-ce-cli=$DOCKER_VERSION containerd.io docker-buildx-plugin docker-compose-plugin ; then
+        msg error "Docker installation failed"
+        exit 1
+    fi
+}
+
+install_lithops()
+{
+    msg info "Install Lithops from pip"
+    if ! pip install lithops==${LITHOPS_VERSION} ; then
+        msg error "Error installing Lithops"
+        exit 1
+    fi
+
+    msg info "Create /etc/lithops folder"
+    mkdir /etc/lithops
+
+    return $?
```

Comment:
```suggestion
    return 0
```

or I think the whole return statement can be just removed :thinking: